pages/[...slug].vue

```diff
@@ -18,6 +18,39 @@
   <EpPageMeta :doc="doc" />
   <NuxtLayout>
     <div class="px-4 print:px-8 mt-10 mb-96 print:mb-0">
+      <div>
+        <nav>
+          <span v-if="breadcrumbs.length != 1">
+            <NuxtLink :to="'/'">
+              <Icon name="heroicons:home-20-solid" size="2em" />
+            </NuxtLink>
+            {{ '>' }}
+          </span>
+
+          <span
```

Comment:
It is better to make v-for element `template`, so there will be no wrapping element in html.

```suggestion
          <template
```